source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -47,6 +47,20 @@ Side note 2. [If you haven't joined the Ember Community on Discord, do so today]
 
 ---
 
+## [Reason #2. Stable and More Reliable Than Ever 💪](https://emberjs.com/ember-community-survey-2019/#MS_Q401)
+
+According to the results of the latest community survey, a large proportion of the community has kept their Ember apps up to date with recent releases. Most Ember apps beyond 3.x were currently locked in at [version 3.7, 3.4 and 3.8](https://emberjs.com/ember-community-survey-2019/#MS_Q401).
+
+Seeing that a great number of Ember apps has managed to keep up with some of the latest is quite striking. It shows that Ember continues to deliver on its promise of **Stability without Stagnation**. As a framework which has been thriving in the fast-paced front-end ecosystem for over seven years, it always promoted itself as a stable and reliable solution for developers who want to make sure that the applications they're building are not only functional, but also still easy to maintain and upgrade tomorrow.
```

Comment:
Jessica, in addition to Preston's suggestion, I might suggest reordering the first sentence so that the sentence is more active. Maybe "It's awesome to see that a great number of Ember apps has kept up with the latest releases."?

A couple of minor updates to your 3rd sentence: "As a framework that has thrived [...], Ember has always promoted itself [...]." I also wonder if the word "survived" would make more sense: "As a framework that has survived the fast-paced front-end ecosystem for over 7 years".

Reply:
Thank you for the suggestion! I updated the first sentence to be a more active voice as well.

In regards to the `thriving` and `surviving` terminology I think it's nice to choose the more uplifting wording to highlight that Ember as a project has indeed not only kept up with the JavaScript community, but has on top of that also proven to be progressive and inspiring to other sub communities in regards to the tools and patterns used. I feel that `thrive` brings that point even better across than `survive` and sounds more empowering. What are your thoughts?

Reply:
Yep, I agree that a positive voice would make your point uplifting and empowering. Let's keep the word thrive as is. :)

Reply:
Awesome, let's do so :-)